config/database.yml

```diff
@@ -21,9 +21,12 @@ default: &default
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: 5
   host: localhost
-  username: root
   database: commons
 
+production:
+  <<: *default
+  username: root
```

Comment:
Yeah, I'm not sure that we need to specify any of this since Heroku uses the `ENV['DATABASE_URL']` value, which includes a username and pw:


> The DATABASE_URL for the Heroku Postgres add-on follows this naming convention:
> `postgres://<username>:<password>@<host>/<dbname>`
 
https://devcenter.heroku.com/articles/heroku-postgresql#spring-java

I've removed the username in fa94c7b. Let's see if it deploys!